Src/RE2.Managed/NativeMethods.cs

```diff
@@ -88,15 +88,36 @@ public static unsafe void GetNamedGroupsSetup(StringUtf8 pattern, int* numCaptur
             }
         }
 
-        public static unsafe bool MatchesNamedGroups(StringUtf8 pattern, StringUtf8 text, GroupNameHeader* groupNameHeaders, byte* groupNamesBuffer, Submatch* submatches)
+        public static unsafe void MatchesNamedGroups2(
+                StringUtf8 pattern,
```

Comment:
>StringUtf8 [](http://example.com/codeflow?start=16&length=10)

Can we replace with String8Interop?

Reply:
Yes with some work, but I would like to push that to a later refactoring task.

Reply:
Added task here: [https://github.com/microsoft/sarif-pattern-matcher/issues/386](https://github.com/microsoft/sarif-pattern-matcher/issues/386)